spec/controllers/idv/enter_password_controller_spec.rb

```diff
@@ -1032,6 +1032,16 @@ def show
     end
 
     context 'user is going through enhanced ipp' do
+      let(:view_context) { ActionController::Base.new.view_context }
+      let(:sp) { build_stubbed(:service_provider, logo: nil) }
+      let(:decorated_sp_session) do
+        ServiceProviderSessionCreator.new(
+          sp: sp,
+          view_context: view_context,
+          sp_session: { issuer: sp.issuer },
+          service_provider_request: ServiceProviderRequestProxy.new,
+        ).create_session
+      end
```

Comment:
Are the added `let` variables here still needed (as well as the `decorated_sp_session` below)?